config/vufind/OpenIDConnectClient.ini

```diff
@@ -0,0 +1,22 @@
+; Open ID Connect authentication settings; see the [Authentication] section of config.ini to enable.
+[OpenIDConnect]
+url = "openidconnect.provider.url"
```

Comment:
Shouldn't this example include the protocol, like  `url = "https://openidconnect.provider.url"`?

Reply:
Fixed